internal/archive/archive.go

```
@@ -41,10 +50,17 @@ func AddDir(archivePath, srcDir, prefix string) error {
 		}
 		data, err := os.ReadFile(path)
 		if err != nil {
-			return fmt.Errorf("read %s: %w", path, err)
+			// Skip unreadable files (broken symlinks, permission errors)
+			return nil
```

golangci-lint (2.12.1)

[error] error is not nil (line 51) but it returns nil

(nilerr)